packages/flex-dev-utils/src/__tests__/keytar.test.ts

```diff
@@ -10,11 +10,11 @@ const inquirer = require('../inquirer');
 // tslint:enable
 
 describe('keytar', () => {
-  const apiKey = 'SK00000000000000000000000000000000';
```

Comment:
All of these changes are because the new serverless upload api only accepts accountSid